natiami/assignment4/FindIslandsTest.cpp

```diff
@@ -0,0 +1,61 @@
+#include"FindIslandsTest.h"
+
+void FindIslandsTest::test(){
+	test_findNumberOfIslands();
+	test_diagonalLands();
+	test_EmptyMapOfTiles();
+	test_allWater();
+	test_allLand();
+}
+
+void FindIslandsTest::test_findNumberOfIslands(){
+	vector<vector<bool> > mapOfTiles{ { false, true, false, true },
+								{ true, true, false, false },
+								{ false, false, true, false },
+								{ false, false, true, false } };
+	findIslands = FindIslands(4, 4, mapOfTiles);
+
+	int expected_nr_island = 3;
+	EXPECT_EQ(expected_nr_island, findIslands.findNumberOfIslands());

```

Comment:
Tip:
If you want, you can make it an (almost) one-liner:
```C++
vector<vector<bool>> mapOfTiles = { ... } ;
EXPECT_EQ(3, FindIslands(mapOfTiles).findNumberOfIslands());  // <-- This is the "one liner".
```

The `EXPECT_EQ(expected, actual)` pattern is very common, so I think it is clear enough that the `3` on the left is the expected result.
However for more complex statements (which does not fit in a line) it is also very clear to put the expected result in a variable named "expected_..." and then comparing it, exactly as you did; so both options are good. Your choice :)